x-pack/plugins/secops/public/components/card_items/index.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  // @ts-ignore
+  EuiCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import numeral from '@elastic/numeral';
+import React from 'react';
+import { pure } from 'recompose';
+
+import { getEmptyTagValue } from '../empty_value';
+
+export interface CardItem {
+  key: string;
+  description: string;
+  value: number | undefined | null;
+}
+
+export interface CardItems {
+  fields: CardItem[];
```

Comment:
Nice type change -- this fixes that typing issue we were seeing and makes things a lot cleaner too 👍

Reply:
Thanks! @XavierM did lots of work on it. It great to see a puzzle solved in different ways!